scripts/release_issue_status/reply_generator.py

```diff
@@ -0,0 +1,110 @@
+        latest_pr_number_int = []
+        for number in latest_pr_number:
+            number = int(re.search('\d+', number).group())
+            latest_pr_number_int.append(number)
+        latest_pr_number_int.sort()
+
+        return latest_pr_number_int[-1]
+
+    def _latest_pr_parse(self):
+        latest_pr = self.rest_specs_object.get_issue(self.latest_pr_number)
+        latest_pr_comments = latest_pr.get_comments()
+        b = [i for i in latest_pr_comments]
+        for comment in latest_pr_comments:
+            if '<h3>Swagger Generation Artifacts</h3>' in comment.body:
+                return self._swagger_generator_parse(comment.body)
+
+    def _swagger_generator_parse(self, context):
+        track1_info_model = ''
+        try:
+            if '<b> azure-sdk-for-python</b>' in context:
+                pattern_python_t1 = re.compile('<b> azure-sdk-for-python</b>.+?</details>', re.DOTALL)
+                python_t1 = re.search(pattern_python_t1, context).group()
+                prttern_python_track1 = re.compile('<ul>\s+?<li>\s+?<a.+</ul>', re.DOTALL)
+                python_track1_info = re.search(prttern_python_track1, python_t1).group()
+                track1_info_model = '<details open><summary><b> python-track1</b></summary>{} </details>'.format(
+                    python_track1_info)
+        except Exception as e:
+            print('track1 generate error')
+        pattern_python = re.compile('<b> azure-sdk-for-python-track2</b>.+?</details>', re.DOTALL)
+        python = re.search(pattern_python, context).group()
+        # the way that reply not contains [Release SDK Changes]
+        # pattern_python_track2 = re.compile('<ul>\s*?<li>\s*?<a.*</ul>', re.DOTALL)
+        pattern_python_track2 = re.compile('<b>track2_.*</ul>', re.DOTALL)
+        python_track2_info = re.search(pattern_python_track2, python).group()
+        track2_info_model = '<details open><summary><b> python-track2</b></summary>{} </details>'.format(
+            python_track2_info)
+        info_model = 'hi @{} Please check the package whether works well and the changelog info is as below:\n' \
+                     '{}\n{}\n' \
+                     '\n* (The version of the package is only a temporary version for testing)\n' \
+                     '\nhttps://github.com/Azure/azure-rest-api-specs/pull/{}\n' \
+            .format(self.issue_object.user.login, track1_info_model, track2_info_model, str(self.latest_pr_number))
+
+        return info_model
+
+    def reply_owner(self, reply_content):
+        self.issue_object.create_comment(reply_content)
+
+    def add_label(self, label_name):
+        if label_name not in self.labels:
+            self.labels.append(label_name)
+        self.issue_object.set_labels(*self.labels)
+
+    def run(self):
+        self.labels = [label.name for label in self.issue_object.labels]
+        self._body_parse()
+        self._whether_change_readme()
+        if not self.whether_change_readme:
```

Comment:
```suggestion
        if not self._whether_change_readme():
```
no need to add `self.whether_change_readme`, just return result directly in `_whether_change_readme`.